internal/oci/runtime_oci.go

```diff
@@ -255,9 +256,9 @@ func prepareExec() (pidFile, parentPipe, childPipe *os.File, err error) {
 	return
 }
 
-func parseLog(log []byte) (stdout, stderr []byte) {
+func parseLog(l []byte) (stdout, stderr []byte) {
```

Comment:
not a fan of this change.  "log" is so much easier to search for than 'l'.

Reply:
I can follow-up on that. `log` shadows the package import so we may rename it to `logBytes` or something else.